example/developerkitgui/developerkitgui.mk

```diff
@@ -2,7 +2,7 @@ NAME := developerkitgui
 
 $(NAME)_SOURCES := developerkitgui.c AliOS_Things_logo.c sensor_display.c
 
-$(NAME)_COMPONENTS := yloop cli board.developerkit.camera_hal board.developerkit.irda_hal
+$(NAME)_COMPONENTS := yloop cli board.developerkit.camera_hal board.developerkit.irda_hal board.developerkit.audio_hal
 
```

Comment:
it seems no need to add audio dependence here when users build developerkitgui app.